editor.planx.uk/src/@planx/components/Pay/Public/Confirm.tsx

```diff
@@ -61,7 +62,10 @@ const PayBody: React.FC<PayBodyProps> = (props) => (
     {!props.error ? (
       <Card>
         <PayText>
-          <Typography variant="h3">
+          <Typography
+            variant="h3"
+            component={props.hideFeeBanner ? "h2" : "h3"}
+          >
```

Comment:
This was a trickier than I anticipated - the banner is integral to the heading heirarchy of the "Pay" page and a11y tests failed when rendering the `MakePayment` implementation with a hidden banner without this adjustment

Reply:
Ah yep, good shout - I hadn't even appreciated that's the same `<Banner />` yet actually - like you suggest below let's pick that one up separately after #1651 :+1: